kumascript/macros/ifmethod.ejs

```diff
@@ -6,13 +6,5 @@ var lang = env.locale;
 var destNewType = lang + '/docs/XPCOM_Interface_Reference/' + $0;
 var destOldType = lang + '/docs/' + $0;
 var apiString = $0 + '.' + $1 + '()';
-if(wiki.pageExists(destNewType)) { /* the page exists from the dekiwiki age */
-    %><code><%-(web.link(wiki.uri(destNewType) + '#' + $1 + '()', apiString))%></code><%
-} else if (wiki.pageExists(destOldType)) { /* the page exists from the mediawiki age */
-    %><code><%-(web.link(wiki.uri(destOldType) + '#' + $1 + '()', apiString))%></code><%
-} else { /* the page doesn't exist */
-  var targeturi = "https://developer.mozilla.org/Article_not_found?uri=" + destNewType +
-        "&ident=" + $0;
-  %><code><a rel="internal" href="<%=targeturi%>" class="new"><%=apiString%></a></code><%
-}
-%>
+
+%><code><%-(web.link(wiki.uri(destNewType) + '#' + $1 + '()', apiString))%></code>
```

Comment:
Ditto. This line is not what used to be on line 10.

Reply:
Ok, I'll have a look now.

Reply:
@peterbe Yesterday you requested changes here and said "just to keep 1" from the below, which I took as just keeping the first `destNewType` declaration which is the `xxx`and removing everything else in 

```ejs
%><code><%-(web.link(wiki.uri(destNewType) + '#' + $1 + '()', apiString))%></code>
<code><%-(web.link(wiki.uri(destOldType) + '#' + $1 + '()', apiString))%></code>

var targeturi = "https://developer.mozilla.org/Article_not_found?uri=" + destNewType +
    "&ident=" + $0;
<code><a rel="internal" href="<%=targeturi%>" class="new"><%=apiString%></a></code>
```

```
if {
  xxx
} else if (cond) {
  yyy
} else {
  zzz
}
```

This was the original `if-else-if` ladder before I introduced any changes (aside from adding `if(true)`)
```ejs
if(true) { /* the page exists from the dekiwiki age */
    %><code><%-(web.link(wiki.uri(destNewType) + '#' + $1 + '()', apiString))%></code><%
} else if (true) { /* the page exists from the mediawiki age */
    %><code><%-(web.link(wiki.uri(destOldType) + '#' + $1 + '()', apiString))%></code><%
} else {
  var targeturi = "https://developer.mozilla.org/Article_not_found?uri=" + destNewType +
        "&ident=" + $0;
  %><code><a rel="internal" href="<%=targeturi%>" class="new"><%=apiString%></a></code><%
}
%>
```

Reply:
It used to say:
```
if (true) {
  %>xxx<%
}
```
you transformed that to:
```
%>xxx
```

no?

Reply:
Yes your correct. I just didn't see a reason to keep the second opening tag `<%` since it won't be including anything else inside. It was there to enclose the closing curly brace `}`  for the next lines `} else if .. ` statement. 

Want me to add it back?

Reply:
Oh I see. The only reason it existed before (the `<%`) was to contain the last `}` character.

Reply:
You got it. Still want it added back in both places? (XULAttr.ejs & ifmethod.ejs)